content/events/seminar-series/_index.md

```diff
@@ -30,8 +30,18 @@ The Nordic-RSE team will provide support and infrastructure.
 
 We will publish upcoming seminar topics and abstracts here as soon as they are confirmed.
 
-Stay tuned!
-
+### Package development in Julia
+- Speaker: [Luca Ferranti](https://lucaferranti.github.io), University of Vaasa
+- **18.08.2021 at 13:00-16:00 CEST** [convert to your timezone](https://arewemeetingyet.com/Stockholm/2021-08-18/13:00)
+
+Julia is constantly gaining popularity both in academia and industry and it is thus an appealing tool for research software engineers. This session will be a hands-on tutorial, which will cover the typical package development workflow in Julia. Topics covered include
+  - Creating a package from scratches
+  - Contributing to existing packages
+  - Tools to test and debug your packages
+  - Tools to document your packages
+
+Moreover, I will share tips and tricks that have helped me making my workflow more efficient and hopefully will help you too.
+The workshop will involve a lot of live coding and you are encouraged to follow along, check the setup instructions here. LINK COMING SOON.
```

Comment:
I don't remember if we discussed it but do we planned to ask attendees to register? It could be a way to "keep" their details (if they want to) for further communication.

Reply:
I don't think we decided the logistics yet, there's a discussion in zulip under `#nordic-rse > Nordic-RSE seminar series`  about this going on. My proposal was to have the zoom(?) link on the webpage (or link to hackmd with zoom link, like for the biweekly meeting) and optional registration to seminar series if someone wants to get reminders for every upcoming event OR just use the nordic-rse newsletter to advertise the seminar events. 

I guess if the talks are not too often, using Nordic-RSE newsletter is ok. If the seminar increases popularities (every 1-2 weeks), then might be good to use a separated newsletter and keep the Nordic-RSE one as low traffic for "more important"(?) announcements.